charts/harvester-cloud-provider/harvester-cloud-provider/0.1.300+up0.1.3/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
```

Comment:
It also seems like the charts changes I'm requesting for in this review might be easier to do in an upstream PR and pulled down... Feel free to add me as a reviewer if that's the path we want to take instead of maintaining patches in this repository.

Reply:
Yes, we will mountain the charts in the [Harvester Chart](github.com/harvester/charts).

Reply:
@aiyengar2 https://github.com/harvester/charts/pull/45. Please help to review the pr.